pkg/volume/local/local.go

```diff
@@ -360,8 +360,7 @@ func (dm *deviceMounter) MountDevice(spec *volume.Spec, devicePath string, devic
 	switch fileType {
 	case hostutil.FileTypeBlockDev:
 		// local volume plugin does not implement AttachableVolumePlugin interface, so set devicePath to Path in PV spec directly
-		devicePath = spec.PersistentVolume.Spec.Local.Path
-		return dm.mountLocalBlockDevice(spec, devicePath, deviceMountPath)
+		return dm.mountLocalBlockDevice(spec, devicePath, spec.PersistentVolume.Spec.Local.Path)
```

Comment:
It should be the second arg that's changed, not the 3rd.

Reply:
Whoops, good catch, thanks! That might explain some of those test failures as well :upside_down_face: